Source/WebCore/css/CSSPageRule.cpp

```diff
@@ -79,8 +79,8 @@ void CSSPageRule::setSelectorText(const String& selectorText)
 String CSSPageRule::cssText() const
 {
     if (auto declarations = m_pageRule->properties().asText(); !declarations.isEmpty())
-        return makeString(selectorText(), " { "_s, declarations, " }"_s);
-    return makeString(selectorText(), " { }"_s);
+        return makeString("@page "_s, selectorText(), " { "_s, declarations, " }"_s);
+    return makeString("@page "_s, selectorText(), " { }"_s);
```

Comment:
```suggestion
    auto selector = selectorText();
    auto optionalSpace = selector.isEmpty() ? ""_s : " "_s;
    return makeString("@page"_s, optionalSpace, selector, " { }"_s);
```
Something like this but share the local variables with the other case too. Or could have 4 different calls to `makeString`, because the code is doing the same thing for `declarations` above.

Reply:
Thanks! I will test it locally now. :-)

Reply:
@darinadler - Thanks for help, pushed newer commit with `NO REGRESSION`, so this is just pure progression patch on WPT.